### PR TITLE
netdata 1.24.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/netdata.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Netdata < Formula
   desc "Diagnose infrastructure problems with metrics, visualizations & alarms"
   homepage "https://netdata.cloud/"
-  url "https://github.com/netdata/netdata/releases/download/v1.23.2/netdata-v1.23.2.tar.gz"
-  sha256 "761aec15901e09f963361752c6e5b5cb723e342a3c5bf6b0624067fceb2ccdfd"
+  url "https://github.com/netdata/netdata/releases/download/v1.24.0/netdata-v1.24.0.tar.gz"
+  sha256 "e7d6691fdce893b5a341fc2ee11b3b4f951e7b426031fae4178e669066c264c0"
   license "GPL-3.0"
```
Comment (Rylan12) [inline Formula/netdata.rb:6]:
```suggestion
  license "GPL-3.0-or-later"
```

ref: https://github.com/netdata/netdata#license

Note that `netdata` redistributes software with different licenses (see https://github.com/netdata/netdata/blob/master/REDISTRIBUTED.md) that may need to be added in a comment for future adoption.

It looks like these are: `MIT`, `BSD-3-Clause`, `BSD-2-Clause`, `CC-BY-4.0 `, `LGPL-2.1-only`, `Apache-2.0`, `WTFPL`, `Unlicense`

Comment (chenrui333) [reply Formula/netdata.rb:6]: All the licenses are permissive licenses, bundled with `GPL-3.0-or-later`, it fits into the relicensing story that I mentioned before.

image

Comment (chenrui333) [reply Formula/netdata.rb:6]: note, `GPLv3` is a strong copyleft license

**Review by dtrodrigues — DISMISSED** [review]

**Comment by Rylan12** [comment]
Canceled because the commit's need to be squashed.

**Comment by dtrodrigues** [comment]
Do they need to be squashed? I've seen it both ways - the license update being in the same commit as the version bump but also when they were left as separate commits. I think separate commits makes more sense since the license change isn't related to the version change.

**Comment by Rylan12** [comment]
Ah, oops. Didn't see this. I generally do by the rule "one commit per formula" but you have a good point. The answer, I guess, is "I'm not sure"

**Comment by chenrui333** [comment]
I think it is fine. Either way should do. Technically I'd prefer to have separate commits to highlight the license changes (it would be good for commit message search, but I can go either way though)

**Comment by chenrui333** [comment]
Thanks @Rylan12! 👍

**Pull request closed without merging**